tests/dbs/warehouse_schema_dump.sql

```diff
@@ -390,84 +392,6 @@ CREATE TABLE takeda (
 	CONSTRAINT takeda_pkey PRIMARY KEY (takeda_trial_id), 
 	CONSTRAINT takeda_meta_id_unique UNIQUE (meta_id)
 );
-CREATE TABLE fdadl (
```

Comment:
These changes were made just because tables were written in different order this time. I will look into it if it occurs again.